bolt-modules/boltlib/lib/puppet/functions/run_plan.rb

```diff
@@ -11,21 +11,21 @@
   # @param args Arguments to the plan. Can also include additional options: '_catch_errors', '_run_as'.
   # @return [PlanResult] The result of running the plan. Undef if plan does not explicitly return results.
   # @example Run a plan
-  #   run_plan('canary', 'command' => 'false', 'nodes' => $targets, '_catch_errors' => true)
+  #   run_plan('canary', 'command' => 'false', 'targets' => $targets, '_catch_errors' => true)
   dispatch :run_plan do
     scope_param
     param 'String', :plan_name
     optional_param 'Hash', :args
     return_type 'Boltlib::PlanResult'
   end
 
-  # Run a plan, specifying $nodes as a positional argument.
+  # Run a plan, specifying $targets as a positional argument.
   # @param plan_name The plan to run.
   # @param args Arguments to the plan. Can also include additional options: '_catch_errors', '_run_as'.
   # @param targets A pattern identifying zero or more targets. See {get_targets} for accepted patterns.
   # @return [PlanResult] The result of running the plan. Undef if plan does not explicitly return results.
   # @example Run a plan
-  #   run_plan('canary', $nodes, 'command' => 'false')
+  #   run_plan('canary', $targets, 'command' => 'false')
```

Comment:
This is going to fail for plans that have a `TargetSpec $targets` parameter since this explicitly passes the `nodes` argument - I think this is probably a bug from #1411  .